src/vanilla/utils/constants.ts

```diff
@@ -1 +1 @@
-export const RESET = Symbol()
+export const RESET = Symbol('RESET')
```

Comment:
This would be acceptable:
```suggestion
export const RESET = import.meta.env?.MODE !== 'production' ? Symbol('RESET') : Symbol()
```

Reply:
just to verify, the `.env.MODE` is used solely by vite, also they compute the `PROD` and `DEV` which are based on the `NODE_ENV` variable.
Besides vite, it seems that no other tools support the `import.meta.env`, e.g. there is open issue for webpack.

Is that ok?